src/js/components/award/SummaryBar.jsx

```diff
@@ -4,21 +4,52 @@
  **/
 
 import React from 'react';
+import moment from 'moment';
 import * as Icons from '../sharedComponents/icons/Icons';
+import InfoSnippet from './InfoSnippet';
 
 const propTypes = {
-    selectedAward: React.PropTypes.object,
-    awardStatus: React.PropTypes.string
+    selectedAward: React.PropTypes.object
 };
 
 export default class SummaryBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillMount() {
+        this.getStatus(this.props.selectedAward);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward !== this.props.selectedAward) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
+        const awardStart = moment(award.period_of_performance_start_date, 'MM-DD-YYYY');
+        const awardEnd = moment(award.period_of_performance_current_end_date, 'MM-DD-YYYY');
+        const current = moment();
+        let progress = "";
+        if (current.isBefore(awardStart)) {
+            progress = "Awarded";
+        }
+        else if (current.isAfter(awardEnd)) {
```

Comment:
Moment's `isBefore()`, `isAfter()`, and `isSame()` functions all allow me to pass a variable dictating the level of granularity for the check (for example, i can set it to day to just compare the day, or I can drill down to minutes and seconds, or up to just the year.)  I've spoken to the UX designer and he's let me know that Start and End date should be inclusive of the current date - if the Start Date is 2/8 and the current date is 2/8, then the status should be `In Progress`.  If the End Date is 2/8 and the current date is 2/8, then the status should be `Complete`.  I'll adjust the logic accordingly with a combo of `isSame` and `isBefore` or `isAfter`.